pkg/pipeline/stream/processor_test.go

```diff
@@ -35,9 +34,8 @@ func TestProcessorNode_Success(t *testing.T) {
 	ctrl := gomock.NewController(t)
 
 	wantRec := record.Record{
-		Position:  []byte(uuid.NewString()),
-		Metadata:  map[string]string{"foo": "bar"},
-		CreatedAt: time.Now().UTC(),
```

Comment:
Do you mean the field `CreatedAt`? It's been moved to metadata and is an optional field, if the connector sets the field it's there, otherwise it's not.

Reply:
Yup, CreatedAt was my question.